eos/b-decays/b-to-psd-nu-nu.cc

```diff
@@ -189,10 +200,14 @@ namespace eos
                 * sqrt_lambda / (power_of<3>(4.0 * M_PI * m_B));
 
             // first term in square brackets in [FLS:2021A], eq. (8)
-            const double contr_vector = lambda / 24.0 * f_p * f_p * std::norm(wc.cL() +  wc.cR());
+            const double contr_vector = lambda / 24.0 * f_p * f_p * std::norm(wc.cVL() +  wc.cVR());
+            // second line in [FLS:2021A], eq. (8) (ignoring the {bs} Wilson coefficients)
```

Comment:
I think I agree with your reading of the formula, but I'm not 100% certain. @mreboud @mbordone can you wade in to confirm or refute that our interpretation of this is correct?